Artifact's model checker doesn't build for kaist-cp-sandbox-01

in the kaist-cp-sandbox-01 server, `opam install --deps-only .` results in:

```
[
[ERROR] The compilation of sail-riscv failed at
        "/kaist-cp-home/jeehoon.kang/.opam/opam-init/hooks/sandbox.sh build make
        LEM_DIR=/kaist-cp-home/jeehoon.kang/.opam/4.10.0/share/lem
        SAIL_DIR=/kaist-cp-home/jeehoon.kang/.opam/4.10.0/share/sail SAIL=sail opam-build".

#=== ERROR while compiling sail-riscv.0.5 =====================================#
# context     2.0.5 | linux/x86_64 | ocaml-base-compiler.4.10.0 | git+https://github.com/rems-project/opam-repository.git#opam2
# path        ~/.opam/4.10.0/.opam-switch/build/sail-riscv.0.5
# command     ~/.opam/opam-init/hooks/sandbox.sh build make LEM_DIR=/kaist-cp-home/jeehoon.kang/.opam/4.10.0/share/lem SAIL_DIR=/kaist-cp-home/jeehoon.kang/.opam/4.10.0/share/sail SAIL=sail opam-build
# exit-code   2
# env-file    ~/.opam/log/sail-riscv-3246283-63107e.env
# output-file ~/.opam/log/sail-riscv-3246283-63107e.out
### output ###
# make ARCH=64 c_emulator/riscv_sim_RV64
# make[1]: Entering directory '/kaist-cp-home/jeehoon.kang/.opam/4.10.0/.opam-switch/build/sail-riscv.0.5'
# mkdir -p generated_definitions/c
# sail -dno_cast -O -Oconstant_fold -memo_z3 -c -c_include riscv_prelude.h -c_include riscv_platform.h -c_no_main model/prelude.sail model/prelude_mapping.sail model/riscv_xlen64.sail model/riscv_flen_D.sail model/prelude_mem_metadata.sail model/prelude_mem.sail model/riscv_types_common.sail model/riscv_types_ext.sail model/riscv_types.sail model/riscv_vmem_types.sail model/riscv_reg_type.sail [...]
# Error:
# SMT solver returned unexpected status 127
# /bin/sh: 1: z3: not found
# make[1]: *** [Makefile:227: generated_definitions/c/riscv_model_RV64.c] Error 1
# make[1]: Leaving directory '/kaist-cp-home/jeehoon.kang/.opam/4.10.0/.opam-switch/build/sail-riscv.0.5'
# make: *** [Makefile:386: opam-build] Error 2
```

The model checker is not built at all.

--- a/README.md
+++ b/README.md
@@ -6,12 +6,13 @@ This supplementary material provides mechanized proofs and a model checker menti
 
 We assume you are on **Ubuntu 20.04**.
 
-### Common requirement
+### Common requirements
 
 ```
-apt install -y build-essential rsync opam
+sudo apt install -y build-essential rsync opam
 opam init
 opam switch create 4.10.0  # or later. If your system OCaml version is >= 4.10.0, you can use it.
+opam update
 ```
 
 ## Mechanized proofs
@@ -130,11 +131,11 @@ We extend the original checker to the PARMv8 model checker by supporting:
 
 ```
 cd model-checker
-sudo apt install findutils libgmp-dev m4 perl pkg-config zlib1g-dev
+sudo apt install -y findutils libgmp-dev m4 perl pkg-config zlib1g-dev z3
 opam repository add rems https://github.com/rems-project/opam-repository.git#opam2
 opam install --deps-only .
-ulimit -s unlimited
-make -j MODE=opt ISA=AArch64
+ulimit -s unlimited  #  Without this, a stack overflow may occur during build.
+make MODE=opt ISA=AArch64
 ```
 
 If it doesn't work, please read `model-checker/README.md` for more details.

